pkg/backup/create_scheduled_backup.go

```diff
@@ -357,7 +357,6 @@ func doCreateBackupSchedules(
 
 	// Create FULL backup schedule.
 	backupNode.AppendToLatest = false
-	backupNode.Options.IncrementalStorage = nil
```

Comment:
yeah, feels fine to squash these commits, given that you're modifying the same files across several commits.